Uchu.StandardScripts/General/Die.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Uchu.Core;
+using Uchu.Core.Client;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.General
+{
+    [ScriptName("l_lego_die_roll.lua")]
+    public class Die : ObjectScript
+    {
+        /// <summary>
+        /// Creates the object script.
+        /// </summary>
+        /// <param name="gameObject">Game object to control with the script.</param>
+        public Die(GameObject gameObject) : base(gameObject)
+        {
+            Listen(gameObject.OnStart, () =>
+            {
+                /*
```

Comment:
Is there a reason this code block for reading the database exists?

Reply:
Line 27 outputs "3000", which I used in line 62 and I forgot to remove the rest of it.